### DISCUSSION TITLE
Setting Authentik to dark mode only

### DISCUSSION THREAD

**Discussion by l4rm4nd** [goauthentik/authentik#7016] [Q&A] [answered]
Authentik's [online documentation](https://goauthentik.io/docs/interfaces/admin/customization#settingsthemebase) references a way to define the default theme. 

I would like to define dark as the default and not use the default automatic option. However, I do not understand where to apply these settings. 

Is this done in some form of JSON/YAML configuration file? Or in the System > Tenants section of Authentik?

![image](https://github.com/goauthentik/authentik/assets/21357789/0fcf4d69-d136-43c2-b3d9-5c8cfc9e35d3)

**Comment by Mase3206** [comment]
It looks like a Python variable, but trying to set that varible in user_settings.py results in a "module not found" error.

**Comment by mahula** [comment]
I am also interested to set another value to the base color scheme than `automatic`.
Where can this be configured?
Please extend the above mentioned documentation section with morespecific detail where this can be confirured and if only in the GUI or as a parameter while deployment.

**Reply by Djeex** [reply]
In admin panel, open the last settings of one of these items
- Brand settings
- Group settings
- User settings

Add
```yaml
settings:
  theme:
    base: light
```
You can change `light` by `dark` or remove all the `theme` section to let authentik use system settings.

**Reply by erictrometer** [reply]
where do you set this exactly? could not find such a section under any of those on version 2025-10:

- Brand settings
- Group settings
- User settings

**Reply by dewi-tik** [reply]
> where do you set this exactly? could not find such a section under any of those on version 2025-10:
> 
>     * Brand settings
> 
>     * Group settings
> 
>     * User settings

You set the values in the attributes field of those setting menus. 

More information is available here: https://docs.goauthentik.io/customize/interfaces/user/

**Reply by l4rm4nd** [reply, marked as answer]
> where do you set this exactly? could not find such a section under any of those on version 2025-10:
> 
> - Brand settings
> - Group settings
> - User settings
> 

https://your-domain.tld/if/admin/#/core/brands

Then under more global settings and attribute:

```yaml
settings:
  theme:
    base: dark
```
